src/hssm/likelihoods/analytical.py

```diff
@@ -322,45 +258,35 @@ def logp_ddm_sdv(
     rt = pt.abs(data[:, 0])
     response = data[:, 1]
     flip = response > 0
-    a = a * 2
+    a = a * 2.0
     v_flipped = pt.switch(flip, -v, v)  # transform v if x is upper-bound response
     z_flipped = pt.switch(flip, 1 - z, z)  # transform z if x is upper-bound response
     rt = rt - t
-    p = ftt01w(rt, a, z_flipped, err, k_terms)
-
-    # This step does 3 things at the same time:
-    # 1. Computes f(t|v, a, z) from the pdf when setting a = 0 and z = 1.
-    # 2. Computes the log of above value
-    # 3. Computes the integration given the sd of v
-    logp = (
-        pt.log(p + epsilon)
-        + (
-            (a * z_flipped * sv) ** 2
-            - 2 * a * v_flipped * z_flipped
-            - (v_flipped**2) * rt
-        )
-        / (2 * (sv**2) * rt + 2)
-        - pt.log(sv**2 * rt + 1 + epsilon) / 2
-        - 2 * pt.log(a + epsilon)
-    )
-    logp = pt.where(rt <= 0, OUT_OF_BOUNDS_VAL, logp)
-    checked_logp = check_parameters(
-        logp,
-        sv >= 0,
-        msg="sv >= 0",
+
+    p = pt.maximum(ftt01w(rt, a, z_flipped, err, k_terms), pt.exp(OUT_OF_BOUNDS_VAL))
```

Comment:
let's rename out_of_bounds_val to out_of_bounds_logp

Reply:
Isn't this a lower bound? Should we make this more specific?

Reply:
yes just to make explicit that this is the lower bound on logp value.